src/v/cloud_storage/recovery_utils.cc

```diff
@@ -55,7 +55,7 @@ ss::future<> place_download_result(
     // The non-empty payload is required for ABS to accept the upload. An empty
     // file results in an authorization exception.
     auto result = co_await remote.upload_object(
-      bucket, cloud_storage_clients::object_key{result_path}, "result", fib);
+      bucket, cloud_storage_clients::object_key{result_path}, "", fib);
```

Comment:
Isn't there a contradiction between the comment and the payload being an empty string?

Reply:
yes missed removing the comment, fixed now.